### ISSUE TITLE
Add option to clear screen

### ISSUE THREAD

**Issue by chintal** [OpenHantek/OpenHantek6022#414]
**Is your feature request related to a problem? Please describe.**
I'm not entirely convinced the trigger is working correctly - specifically, I think some instances are missed. Of course, this could also be because my signal is not actually showing up on every reset. However, once the waveform is on the screen in the normal mode, it just stays there indefinitely. It's always a bit of an effort to get the screen to clear, which I do by switching the trigger mode a few times until it goes away. 

**Describe the solution you'd like**
Have a button somewhere to clear the screen. Regular oscilloscopes include such a button.

**Comment by Ho-Ro** [comment]
> I'm not entirely convinced the trigger is working correctly - specifically, I think some instances are missed. 

You're totally right, b/c the scope has no HW trigger support, triggering works like this:

1. The scope samples free-running one raw buffer (20K samples).
2. It analyses this buffer if the trigger condition is met.
3. If yes, it transfers this buffer to the raw-to-voltage conversion and display. If not, the scope keeps displaying the last triggered view (this is necessary to modify the trace display (V/div, X-Y-position etc). 
4. It goes to 1. and samples the next raw buffer.

The raw sampling is not continuous, if a trigger condition happens in the "blind spots" it is not detected.
The scope is not well suited for fast one-time signals, the best measurement environment are repetitive signal in the extended audio range up to 100 kHz, maybe also 400 kHz from I2C. If you need low-cost analysis better use a Salae clone (~10$) with sigrok, this gives 16 digital channels with up to 16MHz sample rate.
